hazelcast/src/main/java/com/hazelcast/wan/WanReplicationService.java

```diff
@@ -30,7 +31,7 @@
  * to replicate values to other clusters over the wide area network, so it has to deal with long
  * delays, slow uploads and higher latencies.
  */
-public interface WanReplicationService extends CoreService, StatisticsAwareService<LocalWanStats> {
+public interface WanReplicationService extends CoreService, StatisticsAwareService<LocalWanStats>, SharedService {
```

Comment:
I wasn't familiar with `SharedService`. How does that affect the service? 
And how about using `com.hazelcast.spi.NodeEngine#getWanReplicationService` ?

Reply:
`SharedService` is just a marker, has no effect. I wasn't aware of `NodeEngine#getWanReplicationService`, it looks better. I'll change the fix.